### PR TITLE
Update contribution guidelines for approvers

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR updates docs contribution guidelines:
- Removes references to deprecated labels
- Removes section for website admins
- Adds guidelines for approvers

/sig docs
/assign @kbhawkey

### CODE REVIEW

File: content/en/docs/contribute/intermediate.md [hunk 1]
```diff
@@ -109,13 +109,19 @@ more information about the responsibilities of reviewers and approvers, see
   [style guide](/docs/contribute/style/style-guide). Only people listed as
   approvers in the
   [`OWNERS`](https://github.com/kubernetes/website/blob/master/OWNERS) file can
-  approve a PR. To approve a PR, leave an `/approved` comment on the PR.
+  approve a PR. To approve a PR, leave an `/approve` comment on the PR.
```
Comment (kbhawkey) [inline content/en/docs/contribute/intermediate.md]: Should line 115 also be changed, `/approve`?

File: content/en/docs/contribute/participating.md [hunk 2]
```diff
@@ -201,21 +201,29 @@ If you are approved, request that a current SIG Docs approver add you to the
...
+
+- Visit the netlify page preview for a PR to make sure things look good before approving.
 
 #### PR Wrangler
 
-SIG Docs approvers are added to the
+SIG Docs approvers participate in the
 [PR Wrangler rotation scheduler](https://github.com/kubernetes/website/wiki/PR-Wranglers)
-for weekly rotations. All SIG Docs approvers are expected to take part in this
+for weekly rotations. SIG Docs expects all approvers to participate in this
```
Comment (kbhawkey) [inline content/en/docs/contribute/participating.md]: line 225, is it scheduler or schedule?

File: content/en/docs/contribute/intermediate.md [hunk 3]
```diff
@@ -109,13 +109,19 @@ more information about the responsibilities of reviewers and approvers, see
...
 
 A PR is merged when it has both a `/lgtm` comment from anyone in the Kubernetes
 organization and an `/approved` comment from an approver in the
 `sig-docs-maintainers` group, as long as it is not on hold and the PR author
 has signed the CLA.
 
+{{< note >}}
+
+The ["Participating"](/docs/contribute/participating/#approvers) section contains more information for reviewers and approvers, including specific responsibilities for approvers.
```
Comment (kbhawkey) [inline content/en/docs/contribute/intermediate.md]: This note seems to be a duplicate of line 98. Is that intentional?

**Comment by stewart-yu** [comment]
LGTM

File: content/en/docs/contribute/intermediate.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/contribute/intermediate.md]: I figured out the OWNERS, OWNERS_ALIASES files.
I will address the nits that I found in a follow on PR.
Thanks!
/approve

**Pull request merged**